CHANGES/6205.bugfix

```diff
@@ -0,0 +1 @@
+Added check for compatible ``setuptools`` version.
```

Comment:
This is not true anymore, nor is the commit message or the PR title. It needs to reflect the actual effect on the project when the patch is applied.

Reply:
See https://github.com/aio-libs/aiohttp/tree/master/CHANGES#readme for examples.

Reply:
Isn't that what this does still?:
```
requires = [
    "setuptools>=46.4.0",
]
```

Reply:
I changed it to:
```
Added requirement for compatible ``setuptools`` version.
```
Is that better?

Reply:
> Isn't that what this does still?:
> 
> ```
> requires = [
>     "setuptools>=46.4.0",
> ]
> ```

TBF, this specifies a dependency to install, it doesn't do any checks :man_shrugging: